cmd/src/server.rs

```diff
@@ -132,6 +133,12 @@ impl TiKVServer {
 
         // Initialize raftstore channels.
         let (router, system) = fsm::create_raft_batch_system(&config.raft_store);
+        let mut coprocessor_host = Some(CoprocessorHost::new(
+            config.coprocessor.clone(),
+            router.clone(),
+        ));
+        let region_info_accessor = RegionInfoAccessor::new(coprocessor_host.as_mut().unwrap());
+        region_info_accessor.start();
```

Comment:
Why starts it here?

Reply:
I'm trying to create the region_info_accessor before creating GCWorker, so that GCWorker can be build from a region_info_accessor.